test/OpenTelemetry.Collector.Dependencies.Tests/HttpClientTests.cs

```diff
@@ -13,6 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 // </copyright>
+#if NETCOREAPP3_1
```

Comment:
nit: in case of whole file `ifdef` it may be helpful to have `_netcore31` suffix it in the name of the file as well.

Reply:
Done